azure-go-webserver-component/main.go

```diff
@@ -41,6 +41,14 @@ func main() {
 			},
 		})
 
+		subnetID := network.Subnets.Index(pulumi.Int(0)).Id().ApplyT(func(val interface{}) (string, error) {
+			v := val.(*string)
```

Comment:
Total nit, but can this not be `func(v *string)`?

Reply:
Good catch. Updated.